tests/integration/update_cluster/existing_iam/kubernetes.tf

```diff
@@ -0,0 +1,533 @@
+
+  tag = {
+    key                 = "KubernetesCluster"
+    value               = "existing-iam.example.com"
+    propagate_at_launch = true
+  }
+
+  tag = {
+    key                 = "Name"
+    value               = "nodes.existing-iam.example.com"
+    propagate_at_launch = true
+  }
+
+  tag = {
+    key                 = "k8s.io/role/node"
+    value               = "1"
+    propagate_at_launch = true
+  }
+
+  metrics_granularity = "1Minute"
+  enabled_metrics     = ["GroupDesiredCapacity", "GroupInServiceInstances", "GroupMaxSize", "GroupMinSize", "GroupPendingInstances", "GroupStandbyInstances", "GroupTerminatingInstances", "GroupTotalInstances"]
+}
+
+resource "aws_ebs_volume" "a-etcd-events-existing-iam-example-com" {
+  availability_zone = "us-test-1a"
+  size              = 20
+  type              = "gp2"
+  encrypted         = false
+
+  tags = {
+    KubernetesCluster                                = "existing-iam.example.com"
+    Name                                             = "a.etcd-events.existing-iam.example.com"
+    "k8s.io/etcd/events"                             = "a/a"
+    "k8s.io/role/master"                             = "1"
+    "kubernetes.io/cluster/existing-iam.example.com" = "owned"
+  }
+}
+
+resource "aws_ebs_volume" "a-etcd-main-existing-iam-example-com" {
+  availability_zone = "us-test-1a"
+  size              = 20
+  type              = "gp2"
+  encrypted         = false
+
+  tags = {
+    KubernetesCluster                                = "existing-iam.example.com"
+    Name                                             = "a.etcd-main.existing-iam.example.com"
+    "k8s.io/etcd/main"                               = "a/a"
+    "k8s.io/role/master"                             = "1"
+    "kubernetes.io/cluster/existing-iam.example.com" = "owned"
+  }
+}
+
+resource "aws_internet_gateway" "existing-iam-example-com" {
+  vpc_id = "${aws_vpc.existing-iam-example-com.id}"
+
+  tags = {
+    KubernetesCluster                                = "existing-iam.example.com"
+    Name                                             = "existing-iam.example.com"
+    "kubernetes.io/cluster/existing-iam.example.com" = "owned"
+  }
+}
+
+resource "aws_key_pair" "kubernetes-existing-iam-example-com-c4a6ed9aa889b9e2c39cd663eb9c7157" {
+  key_name   = "kubernetes.existing-iam.example.com-c4:a6:ed:9a:a8:89:b9:e2:c3:9c:d6:63:eb:9c:71:57"
+  public_key = "${file("${path.module}/data/aws_key_pair_kubernetes.existing-iam.example.com-c4a6ed9aa889b9e2c39cd663eb9c7157_public_key")}"
+}
+
+resource "aws_launch_configuration" "master-us-test-1a-masters-existing-iam-example-com" {
+  name_prefix                 = "master-us-test-1a.masters.existing-iam.example.com-"
+  image_id                    = "ami-15000000"
+  instance_type               = "m3.medium"
+  key_name                    = "${aws_key_pair.kubernetes-existing-iam-example-com-c4a6ed9aa889b9e2c39cd663eb9c7157.id}"
+  iam_instance_profile        = "kops-custom-master-role"
```

Comment:
I don't see where this is defined.  It's not documented in the TF provider, but I'm guessing this should be the ARN https://www.terraform.io/docs/providers/aws/r/launch_configuration.html#iam_instance_profile

I did notice that the shared tasks didn't appear to use the ARN, so I think this is a genuine issue - I don't think it just applies to terraform.